grammars/javascript.cson

```diff
@@ -1218,6 +1218,31 @@
         ]
       }
       {
+        'begin': '(Relay\\.QL)\\s*(`)'
```

Comment:
Really minor, but should `\\.` be captured as `'meta.delimiter.method.period.js'` so it it styled similarly to method invocations?